assets/javascripts/initializers/lazy-videos.js

```diff
@@ -1,23 +1,46 @@
 import { withPluginApi } from "discourse/lib/plugin-api";
+import getLazyAttributes from "../lib/data-extractor";
+import { hbs } from "ember-cli-htmlbars";
 
 function initLazyEmbed(api) {
-  const lazyVideos = api.container.lookup("service:lazy-videos");
-
   api.decorateCookedElement(
-    (cooked) => {
+    (cooked, helper) => {
+      const lazyContainers = cooked.querySelectorAll(".lazy-video-container");
+
       if (cooked.classList.contains("d-editor-preview")) {
         return;
       }
```

Comment:
would probably do this before the lazyContainers line, as we don't want to do any work in this case apparently